doc/effect_list.md

```diff
@@ -8,974 +8,1104 @@ effect. You can produce these effects ingame by using the `@effect` atcommand.
-886     |  Blue castish cone
-887     |  Wavy sparks
-888     |  Earth Shaker (same as 432)
-889     |  Fast light beams
-890     |  Rotation
-891     |  Magic shots [S]
-892     |  Fastness with hitting sound[S]
-893     |  Blue-white light passing by
-894     |  (Nothing)
-895     |  Big wheel of flat light beams
-896     |  Still sun shaped lightning aura
-897     |  Animated sun shaped lightning aura
-898     |  Animated, curvy sun shaped lightning aura
-899     |  White/red light shots from below
-900     |  Animated, slow curvy sun shaped lightning aura
-901     |  Explosion
-902     |  Floating bedtable texture
-903     |  Castish flamey cone
-904     |  Yellow/pink lights passing by
-905     |  Expanding circle
-906     |  Shield Press (falling shield)
-907     |  Chainy, metalish sound [S]
-908     |  Prestige (sphere of yellow particles)
-909     |  Banding (sphere of red particles)
-910     |  Inspiration (sphere of blue particles)
-911     |  Green castish animation [S]
-912     |  Wall of Thorns unit (green fog cloud)
-913     |  Magic projectiles
-914     |  (Nothing)
-915     |  Crazy Weed
-916     |  Demonic Fire
-917     |  More angry, demonic flames
-918     |  Fire Insignia (demonic flames)
-919     |  Hell's Plant (green snapping plant)
-920     |  Fire Walk unit
-921     |  Vacuum Extreme (whirlwind)
-922     |  Psychic Wave
-923     |  Poison Buster
-924     |  Poisoning animation
-925     |  Some filter
-926     |  Electric Walk unit
-927     |  Earth Grave (speary roots)
-928     |  Ice cloud projectiles
-929     |  Warmer (field of flames)
-930     |  Varetyr Spear (falling spear)
-931     |  (Nothing)
-932     |  Firefly
-933     |  [Client Crash]
-934     |  White, castishly expanding cone
-935     |  Green magic projectile
-936     |  Red, castishly expanding cone
-937     |  Yellow, castishly expanding cone
-938     |  Dark-red, castishly expanding cone
-939     |  Blue, conish aura
-940     |  Snow flake
-941     |  Explosion of red, demonic fire
-942     |  Expanding, white dome
-943     |  Green, fluffy projectile
-944     |  Falling gems
-945     |  (Nothing)
-946     |  Floating lights
-947     |  Blue lightning sphere
-948     |  Two blue lightning spheres
-949     |  Flat, spinning diamond
-950     |  Circling, planetlike spheres
-951     |  Three lightning spheres
-952     |  Flat, spinning gem and two lightning spheres
-953     |  Spinning, planetlike spheres
-954     |  Two lightblue glowing spheres
-955     |  Three spinning flame spheres
-956     |  Flame
-957     |  Spinning planetlike sphere
-958     |  Two flames
-959     |  Purple flame
-960     |  Small yellow explosion
-961     |  Cartoony whirlwind
-962     |  Rising fire
-963     |  Dark filter (like Stone Curse)
-964     |  Same as 920
-965     |  Small white explosion
-966     |  Bone crack
-967     |  Another little explosion
+Effect ID | Constant Name | Description
+ --: | :------------------------------| :----------------------------------
```

Comment:
the table should be properly aligned:
```markdown
  ID | Constant Name                  | Description
----:|:-------------------------------|:----------------------------------
  -1 | EF_NONE                        | None
   0 | EF_HIT1                        | Regular Hit
   1 | EF_HIT2                        | Bash
```

And if you want to use `Effect ID`, the whole first column will have to be enlarged

Reply:
Updated.